app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -269,12 +318,10 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
         mMapFragment?.onLowMemory()
     }
 
-    //TODO: kotlin.UninitializedPropertyAccessException: lateinit property signInButton has not been initialized
-    /**
+    //TODO: com.locoquest.app.MainActivity: 10: Developer console is not set up correctly.
```

Comment:
I went ahead and looked at this TODO item, and solved the issue regarding the error. The next issue looks like it will require the google developer console, which is something I do not have access to for the login.